internal/config/config.go

```diff
@@ -216,6 +219,11 @@ func (cfg *Config) setupKind(name string) {
 	cfg.Kind.AlwaysLoad = true
 }
 
+func (cfg *Config) setupGenericRegistry() {
```

Comment:
I think make this `setupRegistry` and move the logic from line `189` to setup GH as well.